01 Operator Overloading/Operator Overloading.Model/Money.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Operator_Overloading.Model
+{
+    public class Money
+    {
+        private int amount1;
```

Comment:
Why not create public properties instead of creating a backing store property with public get set.